Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+FROM python:3.9.2-slim AS development
+
+WORKDIR /src
+
+RUN pip install pymysql
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
When copying all files you should add a .dockerignore file to exclude stuff like the .git folder.

Reply:
Done; I think I've added all the required things.